lib/reports/cost_report.rb

```diff
@@ -73,6 +73,8 @@ def active_members
     end
 
     def num_volumes
+      return Clusterable::HtItem.all_volumes.count
```

Comment:
This is (and `num_pd_volumes` below) are very inefficient because it will marshal all rows as objects, then count them as an array. We can address this on https://github.com/hathitrust/holdings-backend/pull/335 though